pennylane/io.py

```diff
@@ -407,6 +407,61 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a Qiskit ``NoiseModel`` instance.
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
+        verbose (bool): show a complete list of Kraus matrices for ``qml.QubitChannel`` instead of
+            the number of Kraus matrices and the number of qubits they act on. The default is ``False``.
+        decimal_places (int): number of decimal places to round the Kraus matrices when they are
+            being displayed for each ``qml.QubitChannel`` with ``verbose=False``.
+
+    Returns:
+        qml.NoiseModel: The PennyLane noise model, created based on the input Qiskit
+        ``NoiseModel`` object.
+
+    Raises:
+        ValueError: When a quantum error present in the noise model cannot be converted.
+
+    .. note::
+
+        - This function depends upon the PennyLane-Qiskit plugin, which can be installed following these
+          `installation instructions <https://docs.pennylane.ai/projects/qiskit/en/latest/installation.html>`__.
+          You may need to restart your kernel if you are running it in a notebook environment.
```

Comment:
```suggestion
```

Do we need to say this? Is the motivation just to make sure that the user knows that they need to restart their kernel when they update / install pl-qiskit so that the change takes affect?

Reply:
Yes! It is kind of a safe practice that can help prevent any unwarranted behaviour that comes from working in an unclean environment, especially for registering new entry points if they would be upgrading their `pl-qiskit`.